packages/react-admin/src/mui/form/GenericFormError.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Typography from 'material-ui/Typography';
+import { withStyles } from 'material-ui/styles';
+
+const styles = theme => ({
+    root: {
+        backgroundColor: theme.palette.background.default,
+        padding: 2 * theme.spacing.unit,
+    },
+});
+
+const GenericFormError = ({ translate, error, classes, ...rest }) =>
```

Comment:
Can you add a `className` prop as well and merge it with `classes.root` using `classnames` as we do in other components ?